services/src/main/java/org/exoplatform/officeonline/WOPIService.java

```diff
@@ -425,7 +437,7 @@ public void putFile(EditorConfig config, String lockId, InputStream data) throws
 
         long timeout = System.currentTimeMillis() - versionDate.getTimeInMillis();
         // Version accumulation for same user
-        if (versionable && config.getUserId().equals(versioningUser) && timeout < VERSION_TIMEOUT) {
+        if (versionsAccumulation && versionable && config.getUserId().equals(versioningUser) && timeout < VERSION_TIMEOUT) {
```

Comment:
Add service methods:
```
isVersionAccumulationEnabled()
setVersionAccumulation(boolean)
```
And expose them to REST service scoped to admin users.

Reply:
Updated: https://github.com/exo-addons/office-online/pull/32/commits/8273174fba8747732c8e179f333f5ede33d4c5b2